airbyte-config/init/src/main/resources/config/STANDARD_SOURCE_DEFINITION/c6b0a29e-1da9-4512-9002-7bfd0cba2246.json

```diff
@@ -0,0 +1,8 @@
+{
+    "sourceDefinitionId": "c6b0a29e-1da9-4512-9002-7bfd0cba2246",
```

Comment:
please delete this file or merge your branch with master.

Reply:
Hi @marcosmarxm, I have deleted the file.